include/zephyr/bluetooth/audio/audio.h

```diff
@@ -41,6 +41,9 @@ extern "C" {
 
 #define BT_AUDIO_BROADCAST_CODE_SIZE             16
 
+/** Size of the stream language value, e.g. "eng" */
```

Comment:
maybe a ref to ISO 639-3  (3 ascii chars)?

Reply:
Do we want to add a reference to ISO 639-3 in all places where this is used?

Reply:
Not strictly needed - but a ref would make it very clear why it is always 3

Reply:
In that case we should refer to the assigned numbers document, which is the document that, in this context, define it as 3 octets :) 

I don't disagree that references are nice, but if we want to add references to each defined value here, we should do it across the board, which we can do in a follow up PR